### DISCUSSION TITLE
Server - Error 500 but no log-infos

### DISCUSSION THREAD

**Discussion by gitalexa** [kimai/kimai#2400] [General]
**Describe the bug**
I installed kimai2 on a my hosting environment as described in the documentation. There was no errors. I created the database and inserted the url in .env. 
If i try to open the app, a server error 500 appears and there is no log-file under var/log

**Additional context**
 - Kimai version: 1.13
 - PHP version: 7.4

**Comment by gitalexa** [comment]
added some debug in public/index.php and it crashs on loading:
require dirname(__DIR__) . '/vendor/autoload.php';

**Reply by kevinpapst** [reply]
Check file permissions. In more than 90% of all cases the permissions are causing the problems.
Documentation has infos.

**Reply by gitalexa** [reply]
i set it to 777 - same result

**Reply by kevinpapst** [reply]
there is not "it" in support. there is no magic involved...please provide **detailed information** or help is impossible.
